DXgovernance/davi-monorepo
# Feat: spinner when loading governance type

# Description

Added a loading screen when loading the governance type (while switching between DAOs).

This doesn't affect any functionality, just replaces the previous blank screen and "loading" text with a spinner and the DAVI styles.

![image](https://user-images.githubusercontent.com/75996796/210860893-54a60dde-969b-45c6-ba0d-689fe6517a21.png)


## Type of change


# How Has This Been Tested?

Added snapshot test.

To test it:
 - Change guilds, either via the landing page or going to the URL. The loading spinner should briefly appear.
 - To see it fully, add a `true` condition to the return of the main store in `stores > index`, like: `return true ? (` in line 125.

Closes #56 

# Checklist:
